### PR TITLE
Fix dev

### PR DESCRIPTION
**Pull request by zb32**
## PR checklist

- [ X ] This comment contains a description of changes (with reason).
   Update module directory structure to match nf-core V2.6 and update documentation. 
- [ X ] Make sure your code lints (`nf-core lint`).
- [X ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X ] Usage Documentation in `docs/usage.md` is updated.
- [ X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Include everything till the BUSCO subworkflow (#36).
1. README.md - Address all TODOs: https://github.com/sanger-tol/readmapping/blob/main/README.md
2. Add docs/parameters.md using `nf-core schema docs --help`: https://github.com/sanger-tol/readmapping/blob/main/docs/parameters.md
3. Add docs/output.md: https://github.com/sanger-tol/insdcdownload/blob/dev/docs/output.md

File: README.md
Comment (priyanka-surana) [inline README.md:26]: We do not have a place where the test results can be viewed yet. I think it is best if we delete the last sentence 
> The results obtained from ...

File: README.md
Comment (priyanka-surana) [inline README.md:81]: Need to update credits.
1. Change from `nf-core/blobtoolkit` to `sanger-tol/blobtoolkit`
2. Add your name to the list as well.
3. Give credit to Richard Challis for design/original pipeline and Sujai Kumar for maintaining + Link to the original blobtoolkit
4. Add Matthieu and me
5. Also add a sentence on how Alexander was supported by GSoC
6. Remove TODO

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
1. The usage docs refers to `nf-core/blobtoolkit` and that link, please update
2. In general do a search for similar terms/links throughout the pipeline and update.

File: .github/workflows/ci.yml
Comment (gq1) [inline .github/workflows/ci.yml:25]:
```suggestion
          - ""
```

File: .github/workflows/ci.yml
Comment (priyanka-surana) [inline .github/workflows/ci.yml:24]: I would update this to `22.10.1` at least, but stick to a stable version.
https://github.com/nextflow-io/nextflow/releases

File: README.md
Comment (priyanka-surana) [inline README.md:17]: Either remove Slack, Twitter and YouTube or update it to the correct links. It is all pointing to `nf-core` at this point.

File: assets/email_template.txt
Comment (priyanka-surana) [inline assets/email_template.txt:7]: This should be `sanger-tol`

File: conf/modules.config
Comment (priyanka-surana) [inline conf/modules.config:21]: Was this not in your original `mosdepth` PR?

File: docs/usage.md
Comment (priyanka-surana) [inline docs/usage.md:20]: No spaces in comma separated files. This will cause errors in processing.

File: docs/usage.md
Comment (priyanka-surana) [inline docs/usage.md:33]: Same here. No spaces please. During processing, spaces will get read as part of the file name or data type and cause errors. Does the test samplesheet have spaces?

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Small changes. Mostly looks good 👍

**Review by priyanka-surana — APPROVED** [review]
I think this is all good. I would recommend adding your name as the author as well in `nextflow.config`. Let's get one more approval before we merge.

File: nextflow.config
Comment (priyanka-surana) [inline nextflow.config:166]:
```suggestion
    author          = 'Alexander Ramos, Zaynab Butt'
```

**Pull request merged**
